### PR TITLE
LPS-180924 | Create macro to delete datasets via API

### PR DESCRIPTION
**Pull request by abhnerramos**
**Description**
Create a macro to delete datasets via API.

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-180924

### CODE REVIEW

**Comment by abhnerramos** [comment]
ci:stop

**Comment by abhnerramos** [comment]
ci:test:frontend

**Comment by abhnerramos** [comment]
ci:test:frontend

**Comment by abhnerramos** [comment]
@mateusralv LGTM! Could you check it, please?

**Comment by abhnerramos** [comment]
ci:test:sf
ci:test:frontend

**Comment by mateusralv** [comment]
Wait for https://github.com/liferay-frontend/liferay-portal/pull/3259#issuecomment-1516123504 results... to forward

**Comment by mateusralv** [comment]
All the tests passed...

**Comment by mateusralv** [comment]
ci:test:sf

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/JSONFrontendDatasetEntries.macro [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		if (!(isSet(userPassword))) {
+			var userPassword = "test";
+		}
+
+		var curl = '''
+			${portalURL}/o/c/fdsentries \
+				-u ${userEmailAddress}:${userPassword}
+		''';
+
+		var datasetEntrytIds = JSONCurlUtil.get(${curl}, "$..['id']");
```
[the remark this answers is not in the document]
Comment (mateusralv) [reply modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/JSONFrontendDatasetEntries.macro:50]: Sure! has an extra 't'

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSViews.testcase [hunk 2]
```diff
@@ -28,9 +28,7 @@ definition {
 			PortalInstances.tearDownCP();
 		}
 		else {
-			GoBack();
-
-			FrontendDataSetAdmin.tearDownCP();
+			JSONFrontendDatasetEntries._deleteFDSEntry();
```
[the remark this answers is not in the document]
Comment (abhnerramos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/FDSViews.testcase:31]: ops, you are right! I forgot about that, my bad.

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
Hey, @abhnerramos could you take a look again and let me know what do you think? 😄

**Comment by mateusralv** [comment]
ci:test:frontend-data-set

**Comment by abhnerramos** [comment]
> Hey, @abhnerramos could you take a look again and let me know what do you think? smile

LGTM @mateusralv ! Let's wait for CI analysis.

**Comment by mateusralv** [comment]
ci:forward

**Pull request closed without merging**